packages/editor-ui/src/views/SettingsUsersView.vue

```diff
@@ -51,7 +51,7 @@
 </template>
 
 <script lang="ts">
-import { INVITE_USER_MODAL_KEY, VIEWS } from '@/constants';
+import {EnterpriseEditionFeature, INVITE_USER_MODAL_KEY, VIEWS} from '@/constants';
```

Comment:
Shouldn't there be a space between the curlies and variables according to our code formatting rules?

Reply:
Sorry, auto-imports from WebStorm don't add spaces. Will be all good when we move to prettier though. Fixed!